xbmc/cores/VideoPlayer/DVDCodecs/Overlay/contrib/cc_decoder.c

```diff
@@ -250,7 +268,21 @@ void ccmem_tobuf(cc_decoder_t *dec)
         if (buf->rows[i].cells[l].c != ' ')
           break;
       for (j = f; j <= l; j++)
-        dec->text[dec->textlen++] = buf->rows[i].cells[j].c;
+      {
+        const char* override =
+            charset_overrides[(int)buf->rows[i].cells[j].charset][(int)buf->rows[i].cells[j].c];
```

Comment:
Is it guaranteed that all other array locations will return NULL/nullptr? I'm not sure it is.